importers/djornl/parser.py

```diff
@@ -210,8 +264,15 @@ def process_file(self, file, remap_fn, store_fn, err_list, validator=None):
                     err_list.append(err_msg)
                     continue
 
-            # transform it using the remap_functions
-            datum = self.remap_object(row_object, remap_fn)
+            try:
+                # transform it using the remap_functions
+                datum = self.remap_object(row_object, remap_fn)
+            except Exception as err:
+                err_type = type(err)
+                err_list.append(
+                    f"{file['path']} line {line_no}: error remapping data: {err_type} {err}"
+                )
```

Comment:
prevents mysterious uninformative errors during the remapping of parsed data ==> storable data